.devcontainer/init

```diff
@@ -1 +1,3 @@
+#!/bin/sh
+
```

Comment:
Good catch. Think we also need a `set -e` here?

Reply:
Nvmd, probably redundant -- spun it up and looks like Codespaces will swallow errors and launch a recovery VM if it finds any.